corehq/apps/ota/views.py

```diff
@@ -100,9 +100,15 @@ def search(request, domain):
 def app_aware_search(request, domain, app_id):
     """
     Accepts search criteria as GET params, e.g. "https://www.commcarehq.org/a/domain/phone/search/?a=b&c=d"
+        Daterange can be specified in the format __range__YYYY-MM-DD__YYYY-MM-DD, which is
+            used by App manager case-search feature
```

Comment:
nit: I'd remove "used by app manager case-search feature" as it might be used by something else in the future, and I don't think it is clear what that means (in my mental model, this is used by commcare when doing a case search for date ranges - nothing to do with "app manager")

Reply:
Sure